Cards/MadGraph5_aMCatNLO/NewTests/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8/QCD-4Jets_HT-1000to1200_madgraphMLM-BGenFilter-pythia8.json

```diff
@@ -0,0 +1,20 @@
+{
+    "gridpack_submit": "True",
+    "gridpack_path" : "QCD/QCD-4Jets_HT-1000to1200_madgraphMLM-pythia8",
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/madgraphMLM-hepmcfilter-pythia8.dat", "Filter/LHE_BGenFilterEQ0_filter.dat"],
```

Comment:
swap order `madgraphMLM-hepmcfilter-pythia8` to `madgraphMLM-pythia8-hepmcfilter`